src/osx/textctrl_osx.cpp

```diff
@@ -716,6 +726,16 @@ int wxTextWidgetImpl::GetNumberOfLines() const
     return lines ;
 }
 
+unsigned wxTextWidgetImpl::GetPhysicalLineCount() const
+{
+    return 0;
```

Comment:
@catalinr ,
I don't know anything about how OSX implements stuff.
But I presume that textctrl_osx.cpp is/was just a generic interface for both Carbon and Cocoa.

Now I did implement that functionality in Cocoa, but since I was not sure what this file is I just left it alone.

Reply:
This should be pure virtual or return -1 too. Returning 0 is definitely wrong.